pinax/stripe/models.py

```diff
@@ -648,7 +648,7 @@ class Account(StripeObject):
 
     metadata = JSONField(null=True, blank=True)
 
-    stripe_publishable_key = models.CharField(null=True, blank=True, max_length=100)
+    stripe_publishable_key = models.CharField(null=True, blank=True, max_length=255)
```

Comment:
From the postgresql side there no difference between "varchar" and "text" types, if changed to TextField here we would forget forever about resize migrations, this is just a suggestion

Reply:
That is interesting....
I think for this we are ok. As it happens, Stripe is furture proofing by saying "255" characters....as currently they are only using 107 characters....